great_expectations/validator/metric_configuration.py

```diff
@@ -1,7 +1,5 @@
-from __future__ import annotations
-
 import json
```

Comment:
As a part of creating a clean PR, I pulled over specific parts from Alex's branch here. This was the implementation of the `metric_configuration` that Alex had on his branch. It also seems like the only files changes that might affect the project outside of capital one's contrib location.